tooltester/src/main/java/io/dockstore/tooltester/runWorkflow/WorkflowRunner.java

```diff
@@ -440,20 +445,45 @@ private void setTaskFamily(final List<String> listOfEcsTasksBeforeNewOneWasAdded
         }
     }
 
-    public void uploadRunInfo() {
-        runMetrics.setExecutionStatus(getExecutionStatus());
+    private void uploadRunInfo() {
+        extendedGa4GhApi.executionMetricsPost(new ExecutionsRequestBody().addRunExecutionsItem(runMetrics), Partner.AGC.name(), getEntryNameForApi(), version, "generated with tooltester ('run-workflows-through-wes' command)");
+    }
 
+    private void saveRunInfo() {
+        String fileName = Instant.now().toEpochMilli() + ".json";
+        final String filePath = resultDirectory + "/" + generateKey(getEntryNameForApi(), version, Partner.AGC.name(), fileName);
+        File runFile = new File(filePath);
+        Gson gson = new Gson();
```

Comment:
Should make this a `private static final` constant so it's only instantiated once instead of each time `saveRunInfo()` is called

Reply:
Done